bin/glm

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env node
+
+const mri = require('mri');
+const axios = require('axios');
+const fs = require('file-system');
+require('dotenv').config();
+
+const owner = process.env.OWNER;
+const repo = process.env.REPOSITORY;
```

Comment:
Those info should be automatically inferred from the git configuration of the project and I shouldn't need to provide them.

For instance, assuming that the labels config is in the root project folder, based on that location we can detect the git configuration of the project.